package.json

```diff
@@ -10,10 +10,15 @@
 		"preview": "vite preview",
 		"prepare": "husky install",
 		"test": "npm-run-all test:*",
-		"test:lint": "eslint --ext .js,.ts,.vue --ignore-path .gitignore .",
-		"fix": "npm-run-all fix:*",
+		"test:unit": "echo 'Tests TBD'",
+		"lint": "run-s lint:*",
+		"lint:config": "prettier --check '**/*.{json,yml,yaml}'",
+		"lint:script": "eslint --ext .js,.ts,.vue --ignore-path .gitignore .",
+		"lint:style": "stylelint '**/*.{css,scss,vue}'",
+		"fix": "run-s fix:*",
+		"fix:config": "prettier --write '**/*.{json,yml,yaml}'",
```

Comment:
While I understand the motivation here, we do also have some config in `.js` and `.ts` files, that is why I went for mentioning the file extensions for this particular line explicitly.

Reply:
I'm not sure why it particularly matters in this case (these types of files will almost always hold config while ts and js are more versatile, just as vue files also hold styles), but if you insist I will change it back.

Reply:
I'm thinking about it from the other direction. If I want to fix spacing in my config files, then running `npm run fix:config` maybe won't be enough, even though that command claims to fix the config. Whereas the command `npm run fix:jsonYaml` is more honest about what it is doing. That being said, I don't like the name `fix:jsonYaml` much either and am still open to improvements.

Also, I very much appreciate your work to restructure our scripts, and am grateful that you added that linting script for json and yaml files that I had forgotten :pray:

Reply:
I feel like we might as well name them after the tool: `fix:prettier`, `fix:eslint`, `fix:stylelint`. I’m not sure I would run any of these individually anyways – the `lint-staged` approach seems more convenient (but is there something like `fix-staged`?).

Reply:
> I'm thinking about it from the other direction. If I want to fix spacing in my config files, then running npm run fix:config maybe won't be enough, even though that command claims to fix the config.

Very good point by the way